service/src/main/java/io/aklivity/zillabase/service/internal/handler/ZillabaseAuthUsersHandler.java

```diff
@@ -137,25 +138,25 @@ private byte[] transformRequestBody(
     {
         ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
 
-        try (JsonReader reader = Json.createReader(requestBody);
-             JsonWriter writer = Json.createWriter(outputStream))
+        try
         {
-            JsonObject user = reader.readObject();
-
-            JsonObject request = Json.createObjectBuilder()
-                .add("username", user.getString("username"))
-                .add("email", user.getString("email"))
-                .add("firstName", user.getString("firstName"))
-                .add("lastName", user.getString("lastName"))
-                .add("enabled", true)
-                .add("credentials", Json.createArrayBuilder()
-                    .add(Json.createObjectBuilder()
-                        .add("type", "password")
-                        .add("value", user.getString("password"))
-                        .add("temporary", false)))
-                .build();
-
-            writer.writeObject(request);
+            ZillabaseAuthUserRequest user = jsonb.fromJson(requestBody, ZillabaseAuthUserRequest.class);
+
+            Map<String, Object> request = new HashMap<>();
+            request.put("username", user.username);
+            request.put("email", user.email);
+            request.put("firstName", user.firstName);
+            request.put("lastName", user.lastName);
+            request.put("enabled", true);
+
+            Map<String, Object> credentials = new HashMap<>();
+            credentials.put("type", "password");
+            credentials.put("value", user.password);
+            credentials.put("temporary", false);
+
+            request.put("credentials", List.of(credentials));
+
+            outputStream.write(jsonb.toJson(request).getBytes());
```

Comment:
Can we use `jsonb` for this part as well?
Perhaps a `ZillabaseAuthUserRequestView` class with static `.of(ZillabaseAuthUserRequest)` method setting appropriate fields, then `json.toJson(requestView)` instead of using `Map`, etc.